ui_framework/src/components/header/header_alert/_header_alert.scss

```diff
@@ -0,0 +1,3 @@
+.kuiHeaderAlert {
+
+}
```

Comment:
Can we delete this file and the reference to this class name within the React component? I think it's better to remove no-op stuff like this.

Reply:
good catch. this came from the autogen, but we already had one in the main folder. ill move that one here and delete the other.